core/cmd/local_client_test.go

```
@@ -61,7 +61,7 @@ func TestClient_RunNodeShowsEnv(t *testing.T) {
 		t.Fatal("Timed out waiting for runner")
 	}
 
-	logger.Sync()
+	_ = logger.Sync()
```

Instead of ignoring errors, how about we handle them instead.